src/components/AvatarWithImagePicker.tsx

```diff
@@ -432,7 +397,8 @@ function AvatarWithImagePicker({
             {errorData.validationError && (
                 <DotIndicatorMessage
                     style={[styles.mt6]}
-                    messages={{0: [errorData.validationError, errorData.phraseParam]}}
+                    // eslint-disable-next-line @typescript-eslint/naming-convention
+                    messages={{0: translate(errorData.validationError, errorData.phraseParam as never)}}
```

Comment:
not your fault but this code is confusing. Not sure why `messages` is an object instead of just being an array.